fundingmanager.go

```diff
@@ -3140,6 +3140,14 @@ func (f *fundingManager) handleInitFundingMsg(msg *initFundingMsg) {
 		return
 	}
 
+	// If the caller specified to spend all funds, then we
+	// define the maximum allowed funding amount to be the
+	// current limit of the maximum funding amount.
+	var fundUpToMaxAmt btcutil.Amount
+	if msg.fundMax {
+		fundUpToMaxAmt = MaxFundingAmount
```

Comment:
I just saw that PR #4429 directly affects the maximum funding amount used here. With "wumbo" channels being limitless, one way to handle it could be to just set `fundUpToMaxAmt` to the maximum integer value (`math.MaxInt64`) when "wumbo" feature is enabled. 

To check if "wumbo" is enabled, at this point I think it is necessary to check if "wumbo" is active in both the local and remote feature set (using `msg.peer.LocalFeatures()` and `msg.peer.RemoteFeatures()`).

Reply:
`MaxFundingAmount` is unchanged in that PR.

Reply:
But if both the local and remote nodes support wumbo channels, the flag `fundmax` should fund a channel using _all_ funds, not only up to the existing `MaxFundingAmount`?

Reply:
Yeah, so you'd need to modify `fundUpToMaxAmt` based on if wumbo was set or not as you mentioned.

Reply:
Could also leave this max limit cap for the fundall flag for now, to avoid user by accident making channels using all their wumbo funds.